packages/core/src/rest/ZosmfHeaders.ts

```diff
@@ -32,6 +32,13 @@ export class ZosmfHeaders {
     public static readonly X_IBM_INTRDR_LRECL = "X-IBM-Intrdr-Lrecl";
 
 
+    /**
+     * header indicating recursive = true
+     * @static
+     * @memberof ZosmfHeaders
+     */
+    public static readonly X_IBM_RECURSIVE = {"X-IBM-Option": "recursive"};
```

Comment:
Similar to previous comment - should this object be strongly typed as `IHeaderContent`?